features/lwipstack/mbed_lib.json

```diff
@@ -179,12 +195,16 @@
             "tcpip-thread-stacksize": 8192,
             "default-thread-stacksize": 640,
             "memp-num-tcp-seg": 24,
+            "memp-num-tcpip-msg-inpkt": 32,
```

Comment:
I'm not a huge fan of making these settings per-target - PSOC6 isn't particularly in need of this, it's a particular application. This will increase memory usage for other applications.

But I guess if PSOC6 has plenty of memory to spare, generally increasing the defaults is helpful.

Reply:
Yes I agree, initial thought was application will override these value of default. I will remove it so that PSOC6 chips with low memory does not take a hit.

Reply:
Yes I will remove this change and application will override these values. The PSOC6 with low memory will not take hit as I will remove this from here.